frontend/src/pages/projects/screens/projects/ManageProjectModal.tsx

```diff
@@ -86,8 +86,6 @@ const ManageProjectModal: React.FC<ManageProjectModalProps> = ({
         </Button>,
       ]}
     >
-      <Stack hasGutter>
-        <StackItem>
```

Comment:
I think @andrewballantyne mentioned this in another PR. The Stack adds spacing to elelnts that we are relying on, so i wouldn't remove it.

Is there an a11y issue with the stack?